media/css/mozorg/home/home-firefox-mr106-promo.scss

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-firefox';
+
+.mr106-home-hero {
+    .mzp-c-split-bg {
+        background-color: $color-violet-60;
+    }
+
+    // Override Protocol spacing
+    &.mzp-t-split-nospace {
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+
+    // Override Protocol wordmark
```

Comment:
Side related - we should probably file a protocol issue to update the Firefox word-mark if we're now dropping "browser"?

Reply:
https://github.com/mozilla/protocol-assets/issues/86